.circleci/config.yml

```diff
@@ -13,7 +12,7 @@ jobs:
           command: sudo apt update && sudo apt -y install graphviz
       - run:
           name: Install deps
-          command: pip install --no-cache-dir -r docs/requirements.txt
+          command: pip install --no-cache-dir -r requirements-dev.txt
```

Comment:
why is this change recommended?

Reply:
`requirements.txt` was moved to the root directory because if I'm not mistaken that would be the standard place for it to be. I changed the command to install the dev requirements in order to include the `sphinx` dependency.

Reply:
We intendedly avoid having an explicit `requirements.txt` file at the root, as it is easy to confuse it in purpose with the dependencies listed in the `setup.cfg`.

Unless there's something relevant about it that we should consider, this `docs/requirements.txt` pertains exclusively to the documentation building and I think it would be best to keep it under `docs/`

Reply:
Ah, I see now - my bad, I'm not used to working with `setup.cfg`. I'll fix it.